src/translations/src/cboard.json

```diff
@@ -1,4 +1,5 @@
 {
+  "cboard.components.SelectedCounter.text": "items selected",
```

Comment:
you actually chose the right variation for the property key, using `components` instead of `containers` as other strings, this is something from before a recent refactor, I'll need to fix